CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing to Presto
+
+Presto is an Apache 2.0-licensed open source distributed SQL query engine for running interactive analytic queries against data sources of all sizes, ranging from gigabytes to petabytes.
+
+Presto was designed and written from the ground up for interactive analytics and approaches the speed of commercial data warehouses while scaling to the size of organizations like Facebook.
+
+The community owned and driven Presto project is supported by the [Presto Foundation](https://prestodb.io), an independent nonprofit organization with open and neutral governance, hosted under the [Linux Foundation](https://linuxfoundation.org). Learn more about the [Presto Foundation](https://prestodb.io) here.
+
+All Presto development occurs on [GitHub](https://github.com/prestodb), following open source best practices.
+
+## Technical governance
+
+Presto follows industry best practices, with the technical project and its governance distinctly separate from the Presto Foundation.  The [Technical Charter](https://github.com/prestodb/tsc/blob/master/CHARTER.md) establishes the Presto Technical Steering Committee as the body which oversees all technical aspects of the open source project.  By contrast, the [Presto Foundation](https://prestodb.io) Governing Board is responsible for raising and distributing funds to support project activities, and setting policy.  While the two groups are separate, they work in close coordination to establish, guide, and support activities that lead to a long-term, sustainable future for Presto.
```

Comment:
Many times you have double space before starting new sentences and sometimes not (especially the above section). It would be good to be consistent about that.

Reply:
Thanks! I'll push a patch that fixes this.